about.php

```diff
@@ -2,19 +2,24 @@
 <!DOCTYPE html>
 <html lang="en">
 <?php require_once __DIR__ .'/template/head.phtml';?>
-<title><?php $titleAbout?></title>
+<?php
+$title = $page_titles[basename($_SERVER['SCRIPT_FILENAME'])];
+?>
+<title><?php echo $title; ?></title>
 <body>
    <header>
-       <?php require_once __DIR__ .'/template/menu.phtml';?>
-    <ul class="menu-nav">
-        <?php
-        foreach ($menuitems as $url => $label){
-        ?>
-        <li class="nav-item ">
-            <a href="<?=$url?>" class="nav-link"><?=$label?></a>
-            <?php } ?>
-        </li>
-    </ul>
+       <nav>
+       <ul class="menu-nav">
+           <?php
+           $currentPage = basename($_SERVER['SCRIPT_FILENAME']);

```

Comment:
@ReginaBorisova 
Seems that this code is duplicated on every page. Can it be declared once and re-used?